packages/dart/lib/src/objects/parse_installation.dart

```diff
@@ -85,6 +85,7 @@ class ParseInstallation extends ParseObject {
     set<String?>(keyLocaleIdentifier, ParseCoreData().locale);
 
     //Timezone
+    set<String>(keyTimeZone, _getNameLocalTimeZone());
```

Comment:
Is this change related to the feature "Add include option to getObject and fetch"?

Reply:
No, it is for #616. The title should be changed

Reply:
A PR should only address one issue, unless they are directly related as a consequence of one another. Could you split this into 2 PRs?

Reply:
What title do you suggest? I have no idea

Reply:
> A PR should only address one issue, unless they are directly related as a consequence of one another. Could you split this into 2 PRs?

Sorry, I didn't know about this topic. I separate the PRs

Reply:
No worries, thanks!

Reply:
i reverted to back

Reply:
thanks i'll resolve this